src/VirtoCommerce.CatalogModule.Core/Services/IAssociationService.cs

```diff
@@ -6,7 +6,10 @@ namespace VirtoCommerce.CatalogModule.Core.Services
     public interface IAssociationService
     {
         Task LoadAssociationsAsync(IHasAssociations[] owners);
-
+        Task<ProductAssociation[]> GetAssociationsAsync(string[] ownerIds);
+        Task UpdateAssociationSetAsync(string ownerId, ProductAssociation[] associations);
```

Comment:
Not sure we need ownerId param here. Am I getting right ProductAssociation has the info about the owner?

Reply:
no, it has no references to owner, only to associated product

Reply:
I see, ok then